public/js/question.js

```diff
@@ -34,17 +33,17 @@ $('#mc_answerform').submit(function(evt) {
         url: '/submitanswer',
         data: { questionId: questionId, answer: ans },
         success: function(data) {
-            if (data.result == 'incorrect') {
-                swal('Incorrect', 'Sorry, that\'s the wrong answer', 'error');
-            } else if (data.result == 'correct') {
-                swal({
-                    title: 'Correct',
-                    text: 'Congratulations! You gained ' + data.points + ' points!',
-                    type: 'success'
-                }, function () {
-                    window.location.href = '/home';
-                });
-            }
+            swal({
```

Comment:
Both these functions are almost identical. But I assume theres no way to combine them? Maybe make the success functions run the same function? Not sure of the implications there though.

Reply:
I will try to combine them